tests/BioFSharp.Tests.NetCore/BioFSharp/BioCollections.fs

```diff
@@ -127,6 +127,69 @@ let bioCollectionsTests  =
                     "BioArray.translate did not translate the transcript correctly."
             )
 
+            testCase "isEqual" (fun () ->
+                Expect.equal
+                    (testTranscript |> BioArray.isEqual testTranscript)
+                    0
+                    "BioArray.isEqual did not return correct integer when transcripts were equal."
+            )
+
+            testCase "toString" (fun () ->
+                Expect.equal
+                    (aminoAcidSetArray |> BioArray.toString)
+                    "ACDEFGHIKLMNOPQRSTUVWYXJZB-*"
+                    "BioArray.toString did not return the correct string"
+            )
+
+            testCase "toMonoisotopicMass" (fun () ->
+                Expect.floatClose
+                    Accuracy.high
+                    (testProt |> BioArray.toMonoisotopicMass)
+                    // Masses obtained from University of Washington Proteomics Resource https://proteomicsresource.washington.edu/protocols06/masses.php
+                    (131.04048 + 99.06841 + 113.08406)
+                    "BioArray.toMonoisotopicMass did not return correct mass"
+            )
+
+            testCase "toAverageMass" (fun() ->
+                Expect.floatClose
+                    Accuracy.medium // High accuracy was not passing test
+                    (testProt |> BioArray.toAverageMass)
+                    // Masses obtained from University of Washington Proteomics Resource https://proteomicsresource.washington.edu/protocols06/masses.php
```

Comment:
I also like that you included sources here, so when we have to revisit these tests we have the references in the code.